.github/workflows/nix.yml

```diff
@@ -9,10 +9,10 @@ jobs:
       NIXPKGS_ALLOW_INSECURE: 1
     steps:
     - name: Checkout code
-      uses: actions/checkout@v2
+      uses: actions/checkout@v4
       with:
         submodules: true
-    - uses: cachix/install-nix-action@v13
+    - uses: cachix/install-nix-action@v27
```

Comment:
I reworked the nix stuff here

https://github.com/mirage/ocaml-cohttp/pull/1041

Reply:
Thanks!